README.md

```diff
@@ -7,6 +7,12 @@
 - Ruby
 - Linters: Rubocop
 
+## Getting Started
+
+- `To run the code please use the "Code Runner" VSCode extension`
```

Comment:
### Optional
- [ ] It would be better if we add `git clone https://github.com/Andyveloper/OOP-school-library.git` so that those who don't know how to clone and run the app can appreciate the project since we do not have a live link for this app.  We can not run `ruby main.rb` without cloning the project first. I shall let this pass but kindly make sure to include it before merging it